docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc

```diff
@@ -10,186 +10,239 @@ Before you migrate your databases to the {rhos_long} control plane, retrieve the
 
 .Prerequisites
 
-* Define the following shell variables. Replace the example values with values that are correct for your environment:
+. Define the following shell variables. Replace the example values with values that are correct for your environment:
 +
 ----
-ifeval::["{OpenStackPreviousInstaller}" != "director_operator"]
+$ PASSWORD_FILE="$HOME/overcloud-passwords.yaml"
+$ NAMESPACE="openstack"
 ifeval::["{build}" != "downstream"]
-CONTROLLER1_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100"
-MARIADB_IMAGE=quay.io/podified-antelope-centos9/openstack-mariadb:current-podified
+$ MARIADB_IMAGE=quay.io/podified-antelope-centos9/openstack-mariadb:current-podified
 endif::[]
 ifeval::["{build}" == "downstream"]
-CONTROLLER1_SSH="ssh -i *<path to SSH key>* root@*<node IP>*"
-MARIADB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-mariadb-rhel9:18.0
+$ MARIADB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-mariadb-rhel9:18.0
 endif::[]
-SOURCE_MARIADB_IP=172.17.0.2
-ifeval::["{build}" != "downstream"]
-SOURCE_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
+$ declare -A TRIPLEO_PASSWORDS
+ifeval::["{OpenStackPreviousInstaller}" != "director_operator"]
```

Comment:
This ifeval will have to change. I don't think we've settled on one yet, so I won't put an example here. This is an FYI.